baseapp/testutil/buf.gen.yaml

```diff
@@ -2,4 +2,4 @@ version: v1
 plugins:
   - name: gocosmos
     out: ../..
-    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types
+    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/gogoproto/types/any
```

Comment:
wanted to double check this is optional correct? chains dont have to do this as part of their upgrade?

Reply:
it is optional bc all public types alised, but could be a good idea to use the gogoproto path to not 'depend' on sdk types. But if is ok to use one or the other, I can clarify in the UPGRADING.md that this is indeed optional

Reply:
amazing, thank you!!